### PR TITLE
local k8 deployment

### PR DESCRIPTION
**Pull request by vpethuraj**

### CODE REVIEW

File: deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
```
Comment (gik0geck0) [inline deployment.yaml:1]: Can we move the k8s yaml into the k8s directory?

File: k8s/scripts/run_docker.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x  # show commands being ran
+set -e  # error if anything fails
+set -o pipefail # error on pipe fails too
+
+current_version="latest"
+image_name="dominion1:${current_version}"
```
Comment (gik0geck0) [inline k8s/scripts/run_docker.sh:8]: dominion1 -> dominionstats

File: k8s/scripts/run_docker.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x  # show commands being ran
+set -e  # error if anything fails
+set -o pipefail # error on pipe fails too
+
+current_version="latest"
```
Comment (gik0geck0) [inline k8s/scripts/run_docker.sh:7]: does GCR allow us to overwrite the latest image every time we push?

File: k8s/README.md [hunk 4]
```diff
@@ -1,29 +1,41 @@
 # uisf-foyer on kubernetes
 
-These scripts can be used to deploy foyer + postgres to k8s cluster
+Files in this folder can be used to deploy this LWC app to a k8s cluster.
 
-*To test locally (SAS team):*
-
-## prerequisites
-- Docker Desktop
+## Perequisites
```
Comment (gik0geck0) [inline k8s/README.md:5]: sp: Prerequisites

File: .github/workflows/pipelines.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
+name: Build and Deploy to GKE
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  PROJECT_ID: ${{secrets.GKE_PROJECT}}
+  GKE_CLUSTER: ${{secrets.GKE_CLUSTER NAME}}
```
Comment (gik0geck0) [inline .github/workflows/pipelines.yaml:10]: need an underscore? GKE_CLUSTER_NAME

File: k8s/scripts/run_docker.sh (hunk 3 above)
Comment (vpethuraj) [reply k8s/scripts/run_docker.sh:7]: It'll overwrite the latest image. But it'll keep all previous images as well.

**Review by gik0geck0 — APPROVED** [review]

**Pull request merged**